### PR TITLE
docs: add MAINTAINING.md with maintenance guidelines

### PR DESCRIPTION
**Pull request by zetok**
Most likely doesn't have all the necessary knowledge (yet).

cc people with write access:
@agilob 
@antis81 
@dubslow 
@sudden6 
@TheSpiritXIII 
@tux3 

Should something be changed / clarified?

### CODE REVIEW

**Comment by sudden6** [comment]
could you add the `.gitconfig` line to automatically sign every commit? It is very hard to find elsewhere.

File: MAINTAINING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## SSH
+
+Preferably use SSH.
+
+There are quite a few articles about that:
+https://help.github.com/categories/ssh/
+
+# Commits
+
+- **always** use [commit message format]
```
Comment (SkyzohKey) [inline MAINTAINING.md:34]: Link to CONTRIBUTING.md is missing ;p

Comment (zetok) [inline MAINTAINING.md:34]: Fixed.

File: MAINTAINING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+*“Thou shall GPG-sign.”*
+
+# Git config
+
+## GPG signing
+
+Read it: https://help.github.com/articles/signing-commits-using-gpg/
+
+After you read it, perhaps including "Further reading", do what tip tells you.
```
Comment (sudden6) [inline MAINTAINING.md]: I think there's a `the` missing in `do what tip tells you.`

Comment (zetok) [inline MAINTAINING.md]: Fixed.

File: MAINTAINING.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+**Guidelines, overview of maintenance process, etc.**
+
+*“Thou shall GPG-sign.”*
+
+# Git config
+
+## GPG signing
```
Comment (sudden6) [inline MAINTAINING.md:7]: Maybe point to CONTRIBUTING.md where the setup is already explained?

Comment (zetok) [inline MAINTAINING.md:7]: Yep, when it will be explaining, i.e. on master :)

Comment (sudden6) [inline MAINTAINING.md:7]: Well, now it is on master ;)

Comment (zetok) [inline MAINTAINING.md:7]: Yep, done :)

**Pull request merged**